distribution-core/pom.xml

```diff
@@ -156,6 +156,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>powsybl-computation-local-test</artifactId>
```

Comment:
I wonder if we should be more explicit about the use of containers, in the name.

Reply:
Like powsybl-computation-local-docker-test ?